database/dataloaderSegmentation.py

```diff
@@ -0,0 +1,52 @@
+
+'''
+Creating a Dataloader for segmentation taks
+'''
+
+import numpy as np
+import os
+from PIL import Image
+from torch.utils.data import Dataset
+
+
+class DatasetLoaderSegmentation(Dataset):
+    ''' Derived class to manage Segmentation Dataset Image Loading
+    '''
+
+    def __init__(self, root,
+                 images='images', imagext='png', imageconv='RGB',
```

Comment:
Do we need to have the conversion here. Why is it not enough to be hard-coded.

Reply:
Addition of a variable to contain the conversions